addons/crm_partner_assign/models/crm_lead.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import random
+from odoo import api, fields, models, _
+from odoo.addons.base_geolocalize.models.res_partner import geo_find, geo_query_address
+
+
+class CrmLead(models.Model):
+    _inherit = 'crm.lead'
+    partner_latitude = fields.Float(string='Geo Latitude', digits=(16, 5))
+    partner_longitude = fields.Float(string='Geo Longitude', digits=(16, 5))
+    partner_assigned_id = fields.Many2one('res.partner', string='Assigned Partner', track_visibility='onchange', help="Partner this case has been forwarded/assigned to.", index=True)
+    date_assign = fields.Date(string='Assignation Date', help="Last date this case was forwarded/assigned to a partner")
+
+    def get_interested_action(self, interested):
+        lead_channel_act = self.env.ref('crm_partner_assign.crm_lead_channel_interested_act')
```

Comment:
What if crm_lead_channel_interested_act not found, shouldn't it throw UserError "The CRM Channel Interested Action is missing" ?